src/components/PoolRemove/index.tsx

```
@@ -951,68 +694,34 @@ const PoolRemove = () => {
-          constructor = builder.arc200.tokA;
-          withdrawAmount = Provider_withdraw[0];
-        } else {
-          constructor = builder.arc200.tokB;
-          withdrawAmount = Provider_withdraw[1];
+      do {
+        for (const tok of [tokA, tokB]) {
+          const token = tokens?.find((t) => t.contractId === tok);
+          if (!token) continue;
+          const symbol = token.symbol;
+          const decimals = token.decimals;
+          const constructor =
```

Biome

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)